automation/jenkins_yocto-build-env.sh

```diff
@@ -3,17 +3,13 @@
 set -ev
 
 DOCKERFILE=Dockerfile_yocto-build-env
+REVISION=$(git rev-parse --short HEAD)
```

Comment:
Do you need to export this variable as well? I guess I'm sure where all you use this value. This will also depend on the git context you're in, and I'm not sure how that's structured either. If this is just for building this repo, then it should be fine.

Reply:
I think we are in the workdir when running this script while the repository is cloned in a subdirectory.